### DISCUSSION TITLE
We have Bearer Token that changes after certain time. How we can use your plugin so that Oauth is supported with it.

### DISCUSSION THREAD

**Discussion by satyamtpatel** [grafana/grafana-infinity-datasource#234] [Q&A]
We have bearer token available for API. But it changes after certain time. How we can use your plugin that support OAuth?

**Comment by yesoreyeram** [comment]
Oauth tokens are not supported yet. Any specific api you are interested?

**Comment by satyamtpatel** [comment]
We have our own API that we want to use whose token are getting refresh every 10 minutes. Are you planning to add OAuth support or what are our options?

Thanks,
[https://kyrasolutions.com/wp-content/uploads/2021/10/kyra-solutions-modified-logo.png]<https://kyrasolutions.com/>
Satyam Patel
Director of Technology Development
(850) 345-6798<tel:(877)%20561-5972>
***@***.******@***.***>
www.kyrasolutions.com<https://kyrasolutions.com/>
[Kyra Insights Blog]<https://blog.kyrasolutions.com/>
